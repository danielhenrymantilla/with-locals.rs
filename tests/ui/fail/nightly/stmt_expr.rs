#![feature(proc_macro_hygiene, stmt_expr_attributes)]
include!("../../prelude.rs");

fn stmt ()
{
    #[with]
    print!();
}

fn expr ()
{
    let _ = #[with] expr();
}
