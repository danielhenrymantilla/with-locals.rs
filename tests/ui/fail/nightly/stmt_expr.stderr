error: Missing `#[with]` annotation on the enscoping function
  --> $DIR/stmt_expr.rs:12:13
   |
12 |     let _ = #[with] expr();
   |             ^^^^^^^
   |
   = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: Missing `#[with]` annotation on the enscoping function
 --> $DIR/stmt_expr.rs:6:5
  |
6 |     #[with]
  |     ^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
